shared/cfg/guest-os/Linux/JeOS/22.x86_64.cfg

```diff
@@ -0,0 +1,25 @@
+- 22.x86_64:
```

Comment:
OK, btw after creating the JeOS.23 you should probably also update the `virttest/defaults.py`...